embulk-docs/src/index.rst

```diff
@@ -75,14 +75,18 @@ Documents
 
    customization
```

Comment:
Since the "Customization" section is for developers' topic, it's better to move it in "For developers" section, isn't it?